### DISCUSSION TITLE
How to open a design

### DISCUSSION THREAD

**Discussion by Blebowski** [The-OpenROAD-Project/OpenROAD-flow-scripts#696] [Q&A] [answered]
Hello, 

I have run OpenROAD flow on an example design `aes` with manual build / install of OpenROAD app by myself. Everything finished OK, I can see reports / results. 

How do I open the resulting .db including timing ?

I launched `open_road` and used `read_db` with the "6_final" db, which reads me the database, but that does
not load the constraints, nor the libs I think.

Is there a way how I can open a design after the flow has finished (in GUI), and to the things like "interactive reporting", "clock tree browsing", etc... ? I mean in a way where I load DB, all .lib files, all constraints...

**Comment by maliberty** [comment, marked as answer]
make gui_final
